openjdk-common/src/main/docker/setup-env.d/25-profiler-env.bash

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# configure Cloud Profiler
+
+export PROFILER_AGENT_COMMAND=${PROFILER_AGENT_COMMAND:-"-agentpath:/opt/cprof/profiler_java_agent.so=--logtostderr"}
+
+export PROFILER_AGENT=
+
+if is_true "$PROFILER_ENABLE"; then
+  PROFILER_AGENT=${PROFILER_AGENT_COMMAND}
```

Comment:
We should skip setting this if $JAVA_USER_OPTS contains profiler_java_agent.so string to avoid double-initializing the agent for Alpha users where the instructions to enable the profiler were use JAVA_USER_OPTS.